datadog_lambda/trigger.py

```diff
@@ -0,0 +1,182 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+import base64
+import gzip
+import json
+from io import BytesIO, BufferedReader
+
+
+EVENT_SOURCES = {
```

Comment:
This feels like a mapping rather than a list of event sources. In fact, the mapping has a very clear pattern as well, so I would simply do a `replace("aws:", "")` in `read_event_source` instead.